controller/handler/trial.js

```diff
@@ -59,14 +69,16 @@ function trialView (request, reply) {
                 JOIN stage AS st
                 ON st.id = pa.stageId
                 WHERE st.trialId = ?
-                AND si.endTime > ?
+                AND si.startTime > ?
+                AND si.endTime < ?
                 GROUP BY pa.id
                 `,
                 {
                     type: database.sequelize.QueryTypes.SELECT,
                     replacements: [
                         request.params.id,
-                        startDate.toISOString()
+                        fromDatestr,
+                        toDatestr
```

Comment:
Right, because no default value is being set.
Which is why, defaults should be added in the route.

https://github.com/promisasu/Portal/pull/329#discussion_r58639231